starter-code/apple-pie-starter-code/main.css

```diff
@@ -0,0 +1,145 @@
+html, body, div, span, applet, object, iframe,
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed, 
+figure, figcaption, footer, header, hgroup, 
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+	margin: 0;
+	padding: 0;
+	/*border: 1px red solid;*/
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
+
+h1{
+    font-size: 70px;
+    position: absolute;
+    top: 150px;
+    left: 550px;
```

Comment:
There are many ways to center elements, but usually you are going to use 
```
text-align: center; /* (inline elements) */
```
or 
```
margin: 0 auto; /* (block element) */ 
```
Here you have an amazing guide: https://css-tricks.com/centering-css-complete-guide/